P/Python/build_tarballs.jl

```diff
@@ -22,7 +22,11 @@ apk add zlib-dev libffi-dev
 
 # Create fake `arch` command:
 echo '#!/bin/bash' >> /usr/bin/arch
-echo 'echo i386'   >> /usr/bin/arch
+if [[ "${target}" == *-apple-* ]]; then
+    echo 'echo i386'  >> /usr/bin/arch
+else
+    echo 'echo `echo $target | cut -d - -f 1`'  >> /usr/bin/arch
```

Comment:
If you use double quotes you can basically inline the result
```suggestion
    echo "echo $(echo $target | cut -d - -f 1)"  >> /usr/bin/arch
```
But why not always using `uname -m` instead?

Reply:
When I tried `uname -m` in the apple `runshell` (I presume this is reflective of the CI environment), it gives "x86_64" which is not what cmake wants.